cmd/syncv3/main.go

```diff
@@ -115,31 +115,34 @@ func main() {
 			panic(err)
 		}
 	}
-	h2, h3 := syncv3.Setup(args[EnvServer], args[EnvDB], args[EnvSecret], syncv3.Opts{
-		Debug:                args[EnvDebug] == "1",
-		AddPrometheusMetrics: args[EnvPrometheus] != "",
-	})
 
 	// Initialise sentry. We do this in a separate block to the sentry code below,
-	// because we want to configure logging before starting the pollers—they may want to
-	// log to sentry themselves.
+	// because we want to configure logging before the call to syncv3.Setup---which may
+	// want to log to sentry itself.
 	if args[EnvSentryDsn] != "" {
 		fmt.Printf("Configuring Sentry reporter...\n")
 		err := sentry.Init(sentry.ClientOptions{
-			Dsn:     args[EnvSentryDsn],
-			Release: version,
-			Dist:    GitCommit,
+			Dsn:         args[EnvSentryDsn],
+			Release:     version,
+			Dist:        GitCommit,
+			Environment: "dmr-test",
```

Comment:
Probably not intended to be included? :D

Reply:
Oops!